app/backend/prepdocslib/embeddings.py

```diff
@@ -164,10 +164,17 @@ def __init__(
         open_ai_model_name: str,
         open_ai_dimensions: int,
         credential: Union[AsyncTokenCredential, AzureKeyCredential],
+        open_ai_custom_url: Union[str, None] = None,
         disable_batch: bool = False,
     ):
         super().__init__(open_ai_model_name, open_ai_dimensions, disable_batch)
         self.open_ai_service = open_ai_service
+        if open_ai_service:
+            self.open_ai_endpoint = f"https://{open_ai_service}.openai.azure.com"
+        elif open_ai_custom_url:
+            self.open_ai_endpoint = open_ai_custom_url
```

Comment:
It's only called "base_url" in the OpenAI constructor when its the OpenAI class, otherwise it's called "azure_endpoint". So I'll rename to "azure_endpoint" for consistency with constructor.

Reply:
Actually, our other variables all start with "open_ai" instead of "azure", so I think the current naming is the most internally consistent.